lib/Agrammon/OutputFormatter/JSON.pm6

```diff
@@ -93,10 +93,25 @@ sub make-record($fq-name, $output, $model, $raw-value, $var, :$language, :$print
 sub push-filters(@records, $fq-name, $output, $model,
                  Agrammon::Outputs::FilterGroupCollection $collection,
                  $var) {
+    # model filter groups
+    my %fgs = $model.filter-groups;
```

Comment:
But the filters are really associated with a particular multi module (perhaps occurring in the modules below it).